Soccar [Game-Server]/src/nl/soccar/gameserver/message/PlayerLeaveSessionMessage.java

```diff
@@ -0,0 +1,38 @@
+package nl.soccar.gameserver.message;
+
+import nl.soccar.gameserver.message.handler.PlayerLeaveSessionMessageHandler;
+import nl.soccar.library.enumeration.TeamColour;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = 5, handler = PlayerLeaveSessionMessageHandler.class)
+public class PlayerLeaveSessionMessage extends Message {
+
+    private static final int PLAYER_LEAVE_SESSION_MESSAGE = 5;
+
+    private final String username;
+    private final TeamColour colour;
+
+    public PlayerLeaveSessionMessage(String username, TeamColour colour) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)